docs/guides/scaffolding.md

```diff
@@ -5,98 +5,77 @@ Developing with the {{open_autonomy}} framework usually entails creating all the
+        (SelectKeeperRound, DONE): PrintMessageRound
+        (SelectKeeperRound, NO_MAJORITY): RegistrationRound
+        (SelectKeeperRound, ROUND_TIMEOUT): RegistrationRound
     ```
 
-## Using the scaffold tool: step-by-step instructions
-
-First, create an empty agent using the Open Autonomy CLI. This will create the agent directory, that will contain folders for connections, contracts, protocols and skills.
-```
-autonomy create my_agent
-cd my_agent
-```
-Now that the agent is in place, you can generate the basic structure of a [skill](https://open-aea.docs.autonolas.tech/skill/). For that, you need its [FSM](../fsm.md) specification. In this example, let's copy the contents of the [Hello World](../hello_world_agent_service.md) service FSM specification into a file called ```fsm_specification.yaml```. Put it inside your agent's directory.
-
-```
-alphabet_in:
-- DONE
-- NO_MAJORITY
-- RESET_TIMEOUT
-- ROUND_TIMEOUT
-default_start_state: RegistrationRound
-final_states: []
-label: packages.valory.skills.hello_world_abci.rounds.HelloWorldAbciApp
-start_states:
-- RegistrationRound
-states:
-- PrintMessageRound
-- RegistrationRound
-- ResetAndPauseRound
-- SelectKeeperRound
-transition_func:
-    (PrintMessageRound, DONE): ResetAndPauseRound
-    (PrintMessageRound, ROUND_TIMEOUT): RegistrationRound
-    (RegistrationRound, DONE): SelectKeeperRound
-    (ResetAndPauseRound, DONE): SelectKeeperRound
-    (ResetAndPauseRound, NO_MAJORITY): RegistrationRound
-    (ResetAndPauseRound, RESET_TIMEOUT): RegistrationRound
-    (SelectKeeperRound, DONE): PrintMessageRound
-    (SelectKeeperRound, NO_MAJORITY): RegistrationRound
-    (SelectKeeperRound, ROUND_TIMEOUT): RegistrationRound
-```
-
-Now, run the scaffold tool:
+3. **Generate the template classes.** Using the scaffold tool to generate the skeleton for the classes:
+    ```bash
+    autonomy scaffold fsm my_skill --spec fsm_specification.yaml
+    ```
+    You will see that the generated rounds, payloads and behaviours already appear with their correct names, as well as the `HelloWorldAbciApp` and its transition function.
 
-```
-autonomy scaffold fsm my_skill --spec fsm_specification.yaml
-```
+4. **Push the generated skill to the local registry.** Now that you have successfully created the skill, you can push it to the local registry. Since to fetch all packages from IPFS we had used the remote registry, we now need to reset it so we use a local one during the development:
 
-You will see that the generated rounds, payloads and behaviours already appear with their correct names, as well as the `HelloWorldAbciApp` and its transition function. Now that you have successfully created the skill, you can push it to the local registry. Since to fetch all packages from IPFS we had used the remote registry, we now need to reset it so we use a local one during the development:
-```
-autonomy init --local --reset --author AUTHOR
-```
+    ```bash
+    autonomy init --local --reset --author AUTHOR
```

Comment:
I would not recommend that. The setup is done in the quickstart anyway, and reinitialising to local is then confusing in other guides. We should always assume it is initialised for remote.

Reply:
agree